src/highlight/__tests__/highlight.spec.ts

```diff
@@ -97,12 +94,4 @@ describe('highlight', () => {
     });
     expect(fixture).toMatchSnapshot();
   });
-
-  it('should use `hit.highlighted` if it exists', () => {
-    const fixture = render({
-      attribute: 'highlighted',
-      hit: { highlighted: '<em>foo</em> bar' },
-    });
-    expect(fixture).toMatchSnapshot();
-  });
```

Comment:
I looked into it and IS.js doesn't support this. It seems related to highlighting facet values in a refinement list when its searchable, but even in Angular, it does not work. So I removed the test case.

Reply:
the reason this behaviour still works is because Angular InstantSearch is falling back to the attribute, and the attribute (being the actual highlighted item) is highlight, which is being fallen back to.

see https://github.com/algolia/angular-instantsearch/blob/7c9a1ed15c0f212dcfdaac3ad94522538ea548ad/src/refinement-list/refinement-list.ts#L46

Reply:
You're right, so now the test is the same as the "should fallback to non highlighted when no match".

Not directly related, but I saw that IS.js doesn't have the highlight helper in refinement list items, which differs from this implementation. That could be a nice addition.